resources/qml/UserSettingsPage.qml

```diff
@@ -0,0 +1,218 @@
+// SPDX-FileCopyrightText: 2021 Nheko Contributors
+// SPDX-FileCopyrightText: 2022 Nheko Contributors
+//
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+import "ui"
+import Qt.labs.platform 1.1 as Platform
+import QtQuick 2.15
+import QtQuick.Controls 2.3
+import QtQuick.Layouts 1.2
+import QtQuick.Window 2.13
+import im.nheko 1.0
+
+Rectangle {
+    id: userSettingsDialog
+
+    property bool collapsed: width < 800
```

Comment:
If this '800' is the same collapsed width value we use in the main chat page, it should probably be moved out into a constant of its own in the Nheko global

Reply:
It should ideally depend on the contents, but I couldn't figure out how, so I just measured it by manually shrinking the window .-.